src/status_im/constants.cljs

```diff
@@ -435,6 +435,10 @@
 (def ^:const optimism-goerli-chain-id 420)
 (def ^:const optimism-sepolia-chain-id 11155420)
 
+(def ^:const ethereum "ethereum")
```

Comment:
perhaps better names for these? 🤔

Reply:
maybe a map called `network-names` and extract the `:ethereum`, `:sepolia`, etc keywods?

Reply:
And have a function to extract it network utils maybe

Reply:
Well, the map is already a function to extract its keys